build/release.mk

```diff
@@ -0,0 +1,64 @@
+BUILD_DIR    := ./bin/
+GORELEASER_VERSION := v0.143.0
```

Comment:
can we use v0.154.0? this one is kinda old

Reply:
sure, but check that new one works. We just pin the version to what was used at that time.